transforms/setup-config-with-workspaces/__testfixtures__/basic.input.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  let app = new EmberApp(defaults, {
```

Comment:
Some apps define their options in a variable instead of inline, would these transforms be able to handle this case?

For example:
```js
  const options = {
    emberHighCharts: {
      includeHighCharts: true,
      includeHighStock: false,
      includeHighChartsMore: true,
      includeHighCharts3D: true,
      includeModules: ['solid-gauge'],
    },
  };

  let app = new EmberApp(defaults, options);
  return app.toTree();
```

Reply:
Added a test case for this.